plugins/@grouparoo/iterable/__tests__/.env.example

```diff
@@ -0,0 +1 @@
+ITERABLE_API_KEY = ""
```

Comment:
i usually put an example in here (not the real one of course) but it helps others to know what it should look like.